stylesheets/components/button.css

```diff
@@ -0,0 +1,132 @@
+.btn {
+  display: inline-block;
+  font-weight: 400;
+  text-align: center;
+  user-select: none;
+  border: 1px solid transparent;
+  padding: 0.375rem 0.75rem;
+  font-size: 0.875rem;
+  line-height: 1.25;
+  border-radius: 0.25rem;
+  margin: 0.5rem;
+  color: var(--white-color);
+}
+
+.btn:hover {
+  cursor: pointer;
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-primary:hover {
+  background-color: var(--primary-dark);
+}
+
+.btn-secondary,
+.btn-icon {
+  background-color: var(--secondary-color);
+}
+
+.btn-secondary:hover,
+.btn-icon:hover {
+  background-color: var(--secondary-dark);
+}
+
+.btn-link {
+  background-color: transparent;
+  color: var(--primary-dark);
+  font-weight: 400;
+}
+
+.btn-link:hover {
+  background-color: var(--primary-lighter);
+}
+
+.btn-success {
+  background-color: var(--success-txt);
+}
+
+.btn-success:hover {
+  background-color: var(--success-bg);
```

Comment:
You can give the text some color when you hover over it the text is not clearly visible

Reply:
Adding on this u can also give some darkish green background with the text color while hovering.

Reply:
Thanks guys, I have made the changes! :)